htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -1990,7 +1990,11 @@ public function dispatchProduct($user, $product, $qty, $entrepot, $price = 0, $c
 				if ($product > 0) {
 					// $price should take into account discount (except if option STOCK_EXCLUDE_DISCOUNT_FOR_PMP is on)
 					$mouv->origin = &$this;
-					$result = $mouv->reception($user, $product, $entrepot, $qty, $price, $comment, $eatby, $sellby, $batch);
+					if (!empty($conf->global->RETROCESSION_ALLOW_NEGATIVE_DISPATCH) && $qty < 0) {
```

Comment:
OK Done
Travis tests don't pass anymore ... 
I saw a mistake because of the different orders in the parameteres for livraison() and reception() and normally fixed it
But did not pass again ... (Any idea ?)
A new feature applied since the last test ?